apps/expo/src/app/workout/createNew/index.tsx

```diff
@@ -17,42 +18,15 @@ export default function createNew() {
   return (
     <SafeAreaView style={styles.container}>
       <View className="flex-row justify-evenly h-[60px] items-center">
-        <TouchableOpacity onPress={() => router.back()}>
-            <Text className="text-[20px] text-[#FFFFFF] text-center px-16'">
-              Back
-            </Text>
-        </TouchableOpacity>
-        <Text className="font-koulen text-4xl font-semibold text-[#48476D] text-center px-20">Barbell</Text>
-        <TouchableOpacity>
-          <Text className="text-[20px] text-[#FFFFFF] text-center px-16'">
-            Next
-          </Text>
-        </TouchableOpacity>
-      </View>
-      <View className="mx-3 border-[1px] border-[#CACACA] bg-[#272727] rounded-[5px]">
-        <TextInput
-          className="text-[12px] text-[#CACACA] px-4 py-[6px] mx-1
-          placeholder:text-[#CACACA] placeholder:text-[20px] placeholder:italic" 
-          placeholder= "Search workout by name..."/>
+        <NavBar
+        center={'Exercises'}
+        right={'Next'}
+        />
       </View>
-      <View className="flex flex-row items-center m-2">
-        <TouchableOpacity className="px-4 py-[6px] rounded-[5px] flex-1 mx-1 border-[1px] border-[#CACACA] bg-[#424242]" >
-          <Text className="rounded-[5px] text-[20px] text-[#FFFFFF] text-center">
-            Filter by Equipment
-          </Text>
-        </TouchableOpacity>
-        <TouchableOpacity className="px-4 py-[6px] rounded-[5px] flex-1 mx-1 border-[1px] border-[#CACACA] bg-[#424242]" >
-          <Text className="rounded-[5px] text-[20px] text-[#FFFFFF] text-center">
-            Use Diagram
-          </Text>
-        </TouchableOpacity>
-      </View>
-      <View style={{height: 1, backgroundColor: '#CACACA', marginHorizontal: 13, opacity: 0.3,}}/>
-      <ScrollView className="mb-[25px] py-[5px]">
-        <ExerciseList />
-      </ScrollView>
+
```

Comment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/7100bcdd-e219-493b-8ff5-9f38bfc9dc17)

the filter by equipment and the use diagram need to be synced.

Also we run into errors of

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/07883c59-e1db-434f-b62e-231e8990801f)

I believe Elliot pr fix this. or I could be mistaken, regardless, its an error

Reply:
```
        <TouchableOpacity className="px-4 py-[6px] rounded-[5px] fixed border-[#CACACA] bg-[#424242]" >
            <Text className="rounded-[5px] text-[20px] text-[#FFFFFF] text-center">
              Start Freeform Workout
            </Text>
        </TouchableOpacity>
```

I can't see where this button is at all 

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/6e1680cc-8f99-4e5e-84c3-7c88277cbcaa)